src/payment/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+
+from nurse.models import User
+
+
+class Subscription(models.Model):
+    user = models.OneToOneField(
+        User, on_delete=models.CASCADE, related_name="subscription"
+    )
+    stripe_subscription_id = models.CharField(max_length=255, unique=True)
+    status = models.CharField(max_length=50, null=True)
+    payment_status = models.CharField(max_length=50, null=True)
+    plan = models.BooleanField(default=False)
```

Comment:
Not too sure what this is used for, but I think it deserves an help text, I'm not too sure why this is a boolean or maybe the naming is off, it's confusing me

Reply:
The field name has been replaced with active, which is a boolean used to indicate whether a subscription is currently active or not. This helps in tracking the status of a subscription efficiently